### ISSUE TITLE
Does bleurt support Chinese?

### ISSUE THREAD

**Issue by HuihuiChyan** [google-research/bleurt#11]
I tried to use your fine-tuned model on Chinese, but the result is awful with a 0.5 pearson correlation with sacrebleu. Is it because your model does not support Chinese? If not, then how can I use your codes on Chinese?

**Comment by machelreid** [comment]
Hi @HuihuiChyan! As far as I know BLEURT is (as of right now) is a metric only for English (as shown here:  [https://arxiv.org/pdf/2004.04696.pdf](https://arxiv.org/pdf/2004.04696.pdf)). I don't know if there are any plans to support other languages in the future, but if so, it might be something to look out for! cc: @tsellam

**Comment by HuihuiChyan** [comment]
@machelreid Thanks for your instruction!

**Comment by tsellam** [comment]
@machelreid thanks again for your help!

Indeed--we have no immediate release plan for non-English, but we are looking into it. Please stayed tuned!

**Comment by sailxjx** [comment]
@tsellam can you release the code in the chapter "Generating Sentence Pairs"? I'll try to pre-train the Chinese model.
